CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ if(CCACHE_PROGRAM)
 endif()
 
 # Project macro can only take digits in the version, it splits these out to specific variables
-project(OpenStudio VERSION 3.8.0)
+project(OpenStudio VERSION 3.9.0)
```

Comment:
Please add a pre release tag when you bump like this.

```
set(PROJECT_VERSION_PRERELEASE "alpha")
```

done in dc0ac7cbcba1da5e3010e9a060ce4ecbda1a9be3